packages/blockchain-link/src/workers/base.ts

```diff
@@ -11,11 +11,8 @@ import { WorkerState } from './state';
 import { MESSAGES, RESPONSES } from '../constants';
 import type { Message, Response, BlockchainSettings } from '../types';
 
-// declare worker types locally
-// tsconfig is using "dom" in compilerOptions.
-declare const WorkerGlobalScope: ObjectConstructor | undefined;
+// self is not declared in TS Webworker lib typings
```

Comment:
code history hygiene:
not sure if this blockchain-link change shouldnt be done in separated commit.
This fix was done by changing "web" > "webworker" not because of composite structure

Reply:
done